_posts/2000-01-01-intro.md

```diff
@@ -22,23 +22,23 @@ style: center
 
 See the [release notes page](https://github.com/mockito/mockito/blob/master/doc/release-notes/official.md) and [latest documentation](http://mockito.github.io/mockito/docs/current/org/mockito/Mockito.html).
 
-### Project status
+### Project status updates
 
-* Continuously [delivering new version](https://github.com/mockito/mockito/blob/master/doc/release-notes/official.md) for every git push that passes the build
-* Moving away from [Google Code](https://code.google.com/p/mockito/) to [GitHub](https://github.com/mockito/mockito) (GitHub, you're awesome, thank you!)
-* Releasing [2.0 beta](https://github.com/mockito/mockito/issues/123), 2.0 final when ready
-* Compatible with JDK8 with exception of defender methods, JDK8 support will improve in 2.0
+Project status updates will be announced in various channels, including the [mailing list] <img alt="Google Groups" src="https://groups.google.com/forum/favicon.ico" width="21" height="21"> and [Twitter](https://twitter.com/mockitojava) <img alt="Twitter logo" src="https://g.twimg.com/dev/img/marketing/twitter-for-websites/header-logo.png" width="21" height="21">
```

Comment:
Instead of removing all lines I'd keep a few ones to entice readers to go further. Like

* Mockito 2 imminent, but first an RC
* Mockito 3 scope to be defined, will target java 8